### DISCUSSION TITLE
WHY I AM NOT GETTING ACCESS TO DATA CAMP

### DISCUSSION THREAD

**Discussion by Manasvipasala** [community/community#70675] [GitHub Education]
### Select Topic Area

Question

### Body

i have access for github student but while i am trying to open data science and machine learning in data camp it was not showing option for me

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Manasvipasala , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by Gyanendu01** [comment]
### Clear Browser Cache:
Sometimes, browser cache can cause issues with website functionality. Try clearing your browser cache and cookies, then log in again.

### Check Course Availability: 
Verify that the specific courses related to data science and machine learning on DataCamp are part of the GitHub Student Developer Pack offer. It's possible that not all courses on DataCamp are included in the offer, so check the details of the offer to see which courses are available.

### Update Your Profile:
 Check your profile settings on DataCamp to ensure that you have correctly associated your GitHub Student status. Sometimes, this step is necessary to link your GitHub account with the educational resources.

### Resubmit Documentation (if needed):
 If you haven't provided all the required documentation to verify your student status, make sure you do so. Check if DataCamp requires additional verification steps.

**Reply by queenofcorgis** [reply]
Hi @Gyanendu01 , 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.
